docs/source/development/tools/xtce_generator.rst

```diff
@@ -15,31 +23,29 @@ called `xtce_generator_yourinstrument.py`.
 An example of how to use the script is `xtce_generator_codice.py` which is also
 located in the `tools/xtce_generation` directory.
 
-Prerequisites
--------------
-
-Before you begin, ensure you have the following:
+Before you Start
+----------------
 
-- Python installed (version 3.9 or higher)
-- The required Python packages installed:
-    - pathlib
-    - tools.xtce_generation.telemetry_generator
+Generating XTCEs is only done whenever packet definitions get updated, and thus it
+is not a part of the main processing package. To use it there are a few extra
+dependencies like `pandas` that you can install with
+    poetry install --extras tools
```

Comment:
Can you make this either a code block (with `::`) or make it monospaced (with two tick marks ``)